### PR TITLE
Compiler and frc year updates for 2019

### PR DESCRIPTION
**Pull request by ThadHouse**
These should be the last changes needed from my side, other then the wpilib versions. We can leave this PR open until jenkins is publishing everything and we can add those.

### CODE REVIEW

File: src/main/groovy/edu/wpi/first/gradlerio/wpi/WPIExtension.groovy [hunk 1]
```diff
@@ -34,6 +34,9 @@ class WPIExtension {
 
     String frcYear = '2019'
 
+    boolean checkImage = true
```
Comment (JacisNonsense) [inline src/main/groovy/edu/wpi/first/gradlerio/wpi/WPIExtension.groovy]: Is there a reason these shouldn't be in the Artifact?

Comment (ThadHouse) [reply src/main/groovy/edu/wpi/first/gradlerio/wpi/WPIExtension.groovy]: tbh I totally blanked that roborio was a modifyable easily class. fixing

File: src/main/groovy/edu/wpi/first/gradlerio/wpi/toolchain/install/LinuxToolchainInstaller.groovy [hunk 2]
```diff
@@ -1,23 +1,59 @@
...
+            d.dest dst
+            d.overwrite false
+        }
+        da.execute()
+        if (da.upToDate) {
+            println "Already Downloaded!"
+        }
+
+        println "Extracting..."
+        File extrDir = new File(WPIToolchainPlugin.toolchainExtractDirectory(), "linxu")
```
Comment (JacisNonsense) [inline src/main/groovy/edu/wpi/first/gradlerio/wpi/toolchain/install/LinuxToolchainInstaller.groovy]: linxu

**Comment by JacisNonsense** [comment]
This will close #143

File: src/main/groovy/edu/wpi/first/gradlerio/wpi/toolchain/install/LinuxToolchainInstaller.groovy (hunk 2 above)
Comment (ThadHouse) [reply src/main/groovy/edu/wpi/first/gradlerio/wpi/toolchain/install/LinuxToolchainInstaller.groovy]: Fixed.

**Pull request merged**